src/Service.Tests/SqlTests/GraphQLMutationTests/MultipleCreateMutationTests/MsSqlMultipleCreateMutationTests.cs

```diff
@@ -0,0 +1,698 @@
+                                                    }
+                                                  }
+                                                ]
+                                              },
+                                              ""authors"": {
+                                                ""items"": [
+                                                  {
+                                                    ""id"": 5001,
+                                                    ""name"": ""Author #1"",
+                                                    ""birthdate"": ""2000-01-02""
+                                                  },
+                                                  {
+                                                    ""id"": 5002,
+                                                    ""name"": ""Author #2"",
+                                                    ""birthdate"": ""2001-02-03""
+                                                  }
+                                                ]
+                                              }
+                                            },
+                                            {
+                                              ""id"": 5002,
+                                              ""title"": ""Book #2"",
+                                              ""publisher_id"": 1234,
+                                              ""publishers"": {
+                                                ""id"": 1234,
+                                                ""name"": ""Big Company""
+                                              },
+                                              ""reviews"": {
+                                                ""items"": []
+                                              },
+                                              ""authors"": {
+                                                ""items"": [
+                                                  {
+                                                    ""id"": 5003,
+                                                    ""name"": ""Author #3"",
+                                                    ""birthdate"": ""2000-01-02""
+                                                  },
+                                                  {
+                                                    ""id"": 5004,
+                                                    ""name"": ""Author #4"",
+                                                    ""birthdate"": ""2001-02-03""
+                                                  }
+                                                ]
+                                              }
+                                            }
+                                          ]
+                                        }";
+
+            await ManyTypeMultipleCreateMutationOperationRelationshipsDefinedInConfig(expectedResponse);
```

Comment:
How are you ensuring there is no FK constraint between say entities like Book/Author for these tests where you say the relationship is only defined in config file?

Reply:
Also, there are entities with custom relationship added in the mutation order determination PR. For the cases where the mutation order is determined successfully, can you add tests which confirm the request executed successfully right through till query execution?

Reply:
I've introduced new set of entities, tables (without foreign key references in any of them) and relationship definition for the scenario where relationship is configured only through config file.

New entities:
`Publisher_MM`, `Book_MM`, `Review_MM`, `Author_MM`, `WebsiteUser_MM`. 

Considering that we need to test out all relationship types and multiple levels, introducing new entities and tables this way makes it easier to understand the relationship types between these entities (No extra effort needed to familiriaze with new set of entities and how each of them relate with each other). The only difference here is that there exists no foreign key definition in the database layer. All the relationships are defined only through config file.